src/plugins/info/plugin.py

```diff
@@ -103,6 +110,57 @@ def on_config(self, config):
             log.error("Please remove 'hooks' setting.")
             self._help_on_customizations_and_exit()
 
+        # Assure that possible relative paths, which will be validated
+        # or used to generate other paths are absolute.
+        config.config_file_path = _convert_to_abs(config.config_file_path)
+        config_file_parent = os.path.dirname(config.config_file_path)
+
+        # The theme.custom_dir property cannot be set, therefore a helper
+        # variable is used.
+        custom_dir = config.theme.custom_dir
+        if custom_dir:
+            custom_dir = _convert_to_abs(
+                custom_dir,
+                abs_prefix = config_file_parent
+            )
+
+        # Support projects plugin
+        projects_plugin = config.plugins.get("material/projects")
```

Comment:
This is only for path validation, right? I tested packing up our examples repository and commenting out the projets plugin and packing still works. This could be made a little clearer in the comment, because at first I thought that the info plugin explicitly only supports packing up multi-project folders if the projects plugin is enabled.

Reply:
Not just path validation, but also to be able to dynamically exclude the `site_dir` in the projects plugin. 
When I first wrote it I didn't want to use the `sitemap.xml` file to detect site directories, but ultimately decided to do that for the multi-language directory.

I don't think that the option to disable projects file gathering when the projects plugin is disabled is worth pursuing.

Instead to minimise the code churn, I'll just remove it and the projects site directories will be excluded just like the community multi-language projects.

Good catch 👌

Reply:
I'm sorry, but I don't understand what you mean with your last sentence (in store?)

Reply:
Should be "in-store" I guess. I meant that I don't know what are the plans for projects plugin and sitemap manipulation. 
As per the current limitations: https://squidfunk.github.io/mkdocs-material/plugins/projects/#limitations

In the future the projects plugin will somehow manage the sitemap generation, so perhaps there will be an option to disable sitemap creation (?). This is the edge-case dimension all over again 😅 

If you're sure that the current behaviour will be respected and there will always be a `sitemap.xml.gz` file either in every site directory or the global projects site directory, then I could remove the dynamic exclusion part and just keep the exclusion based on the `sitemap.xml.gz` for the build directories.

Reply:
Ah, I see, thanks for clarifying! Yes, we might change the way how the sitemap is generated to allow for a better consolidated experience, thus it might be possible that nested `site_dir` folders do not contain sitemaps. However, nothing to worry about right now.